slides/src/content/directives.md

```diff
@@ -246,9 +236,9 @@ export class AppComponent {
 
 ## `NgFor` Directive
 
-- Mechanism to define multiple chunks of UI at once based on an iterable
-- The internal variable of the iteration (`item`) is scoped in the template
-- Similar syntax of a `for...of` statement
+- Mechanism to define multiple chunks of UI at once based on an iterable <!-- .element: class="smaller-font" -->
+- The internal variable of the iteration <!-- .element: class="smaller-font" --> (`item`) is scoped in the template <!-- .element: class="smaller-font" -->
+- Similar syntax of a <!-- .element: class="smaller-font" --> `for...of` statement <!-- .element: class="smaller-font" -->
```

Comment:
To avoid reducing the size I would change the list like this:

> - Mechanism to define multiple chunks of UI at once based on an iterable using a `for...of` syntax
> - The internal variable of the iteration (`item`) is scoped in the template

That will give you and extra vertical space